.code-samples.meilisearch.yaml

```diff
@@ -363,6 +363,10 @@ search_parameter_guide_matching_strategy_2: |-
   client.index('movies').search('big fat liar', {
     matchingStrategy: 'all'
   })
+search_parameter_guide_matching_strategy_3: |-
+  client.index('movies').search('big fat liar', {
+    matchingStrategy: 'frequency'
+  })
```

Comment:
```suggestion
```

Reply:
Sorry I remove this because I don't know yet what documentation will ask from us, I will update it myself, don't worry
Thanks for helping